src/Agent/NewRelic/Agent/Core/Api/AgentApi.cs

```diff
@@ -336,16 +336,16 @@ void work()
         /// Only the first 1000 characters are retained.
         /// </param>
         /// <param name="value">The numeric value to add to the current transaction.</param>
-        [Obsolete("Will be dropped in a future version.  Use Transaction.AddCustomAttribute instead")]
+        [Obsolete("This method has been stubbed out and just logs a warning.  Use Transaction.AddCustomAttribute instead")]
```

Comment:
Do we want to more explicitly call out that this will do nothing?

Reply:
Will do

Reply:
Done